Makefile

```diff
@@ -290,7 +306,11 @@ docs-other:
 
 .PHONY: docs-serve
 docs-serve: docs ## Run the aurae.io static website locally
+ifeq ($(uid), 0)
+	./hack/serve.sh
+else
 	sudo -E ./hack/serve.sh
```

Comment:
this shouldn't need to be root i think?

Reply:
I'm having trouble testing this as it needs protoc-gen-doc and I can't seem to get it correctly installed.